### PR TITLE
UI Added to Dashboard Generator for Evolution Bucket Index

### PR DESCRIPTION
**Pull request by ab0092**

### CODE REVIEW

File: dashboard-generator/generator.js [hunk 1]
```diff
@@ -317,7 +317,7 @@ window.addEventListener('load', function () {
 
   function updatePostData() {
     const BASE_URL = 'https://telemetry.mozilla.org/';
-
+
```
Comment (chutten) [inline dashboard-generator/generator.js]: this whitespace appears to be unintentionally included.

File: dashboard-generator/index.html [hunk 2]
```diff
@@ -102,6 +102,10 @@ <h3>Welcome to the Telemetry Dashboard Generator!</h3>
           <input type="number" id="evo-versions" value="1"/>
           <div class="help">Over how many version back from the selected version would you like to see the values evolve?</div>
         </label>
+        <label>Evolution Bucket Index:
+          <input type="number" id="evo-bucketindex" value="0"/>
+          <div class="help">Which bucket index would you like to choose for enumerated histograms</div>
```
Comment (chutten) [inline dashboard-generator/index.html]: Add a "?" at the end to match style with the `#evo-versions` help text.

**Comment by ab0092** [comment]
@chutten i have made the required changes. I think now we are good to merge.

**Review by chutten — APPROVED** [review]

**Comment by chutten** [comment]
And merged it has now been. This fixes #572

**Pull request merged**